src/core/utils.js

```diff
@@ -563,6 +563,9 @@ export const sorters = {
   operationsSorter: {
     alpha: (a, b) => a.get("path").localeCompare(b.get("path")),
     method: (a, b) => a.get("method").localeCompare(b.get("method"))
+  },
+  apisSorter: {
+    alpha: (a, b) => a.getIn([0, "operation", "tags", 0]).localeCompare(b.getIn([0, "operation", "tags", 0]))
```

Comment:
@shockey "0" means the first item.
`getIn([0, "operation", "tags", 0])` is a short path for `.get(0).get('operation').get('tags').get(0)`, but I can use the long path if it makes it clearer for you.